src/truecallerpy/login.py

```diff
@@ -73,7 +72,7 @@ async def login(phone_number: str) -> dict:
         "clientsecret": "lvc22mp3l1sfv6ujg83rd17btt",
     }
     try:
-        async with httpx.AsyncClient(proxies=PROXY, verify=SSL_CONTEXT) as client:
+        async with httpx.AsyncClient() as client:
```

Comment:
I may not have the context on this, but why are we not removing the PROXY from verify_otp in this PR as well? Can you add context to the PR if there's a reason for it?

Reply:
There isn't. We should and I forgot to. Will fix

Reply:
Good catch

Reply:
Ok I made an update - I decided to keep the cache but remove the SSL stuff 

In a day or two we can see if the proxy is causing the timeout issues I saw. If it is I will remove the proxy and we can use VPN